Lib/tempfile.py

```diff
@@ -289,9 +294,13 @@ def gettempdir():
             _once_lock.release()
     return tempdir
 
+def gettempdir():
+    """A string version of tempfile._gettempdir()."""
```

Comment:
don't mention the internal function, users shouldn't know that it exists.  `"""Returns tempfile.tempdir as a str."""`